citations-tool/tool/src/webapp/js/citationscript.js

```diff
@@ -671,11 +671,34 @@ function checkRequiredFields( alertMsg ) {
     return false;
   }
   
+  //If electronic citation check if at least one url is supplied
+  if($('#type_selector').val() == 'electronic'){
+      var showUrlAlert = $("input:text[name^='url_']" ).filter(function(){
+                         return this.value.trim().length
+                         }).length > 0;
+      if(!showUrlAlert){
```

Comment:
Seems silly that someone can add a URL to the second URL box and leave first empty!